var/spack/repos/builtin/packages/py-imagehash/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyImagehash(PythonPackage):
+    """A Python Perceptual Image Hashing Module"""
+
+    homepage = "https://github.com/JohannesBuchner/imagehash"
+    pypi = "ImageHash/ImageHash-4.3.1.tar.gz"
+
+    maintainers = ["thomas-bouvier"]
+
+    version("4.3.1", sha256="7038d1b7f9e0585beb3dd8c0a956f02b95a346c0b5f24a9e8cc03ebadaf0aa70")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-pillow", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("pil", type=("build", "run"))
```
We have a virtual package for pillow that lets you used pillow-simd instead.